Documentation/gettingstarted/host-firewall.rst

```diff
@@ -0,0 +1,156 @@
+It allows communications from outside the cluster only on port TCP/22. All
+communications from the cluster to the hosts are allowed.
+
+Host policies don't apply to communications between pods or between pods and
+the outside of the cluster, except if those pods are host-networking pods.
+
+.. literalinclude:: ../../examples/policies/host/demo-host-policy.yaml
+
+To apply this policy, run:
+
+.. parsed-literal ::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/policies/host/demo-host-policy.yaml
+    ciliumclusterwidenetworkpolicy.cilium.io/demo-host-policy created
+
+The host is represented as a special endpoint, with label ``reserved:host``, in
+the output of command ``cilium endpoint list``. You can therefore inspect the
+status of the policy using that command.
+
+.. parsed-literal ::
+
+    $ cilium endpoint list
+    ENDPOINT   POLICY (ingress)   POLICY (egress)   IDENTITY   LABELS (source:key[=value])                       IPv6                 IPv4           STATUS
+               ENFORCEMENT        ENFORCEMENT
+    266        Disabled           Disabled          104        k8s:io.cilium.k8s.policy.cluster=default          f00d::a0b:0:0:ef4e   10.16.172.63   ready
+                                                               k8s:io.cilium.k8s.policy.serviceaccount=coredns
+                                                               k8s:io.kubernetes.pod.namespace=kube-system
+                                                               k8s:k8s-app=kube-dns
+    1687       Disabled (Audit)   Disabled          1          reserved:host                                                                         ready
+    3362       Disabled           Disabled          4          reserved:health                                   f00d::a0b:0:0:49cf   10.16.87.66    ready
+
+
+Adjust the Host Policy to Your Environment
+==========================================
+
+As long as the host endpoint is running in audit mode, communications
+disallowed by the policy won't be dropped. They will however be reported by
+``cilium monitor`` as ``action audit``. The audit mode thus allows you to
+adjust the host policy to your environment, to avoid unexpected connection
+breakages.
+
+.. parsed-literal ::
+
+    $ cilium monitor -t policy-verdict --related-to $HOST_EP_ID
+    Policy verdict log: flow 0x0 local EP ID 1687, remote ID 6, proto 1, ingress, action allow, match L3-Only, 192.168.33.12 -> 192.168.33.11 EchoRequest
+    Policy verdict log: flow 0x0 local EP ID 1687, remote ID 6, proto 6, ingress, action allow, match L3-Only, 192.168.33.12:37278 -> 192.168.33.11:2379 tcp SYN
+    Policy verdict log: flow 0x0 local EP ID 1687, remote ID 2, proto 6, ingress, action audit, match none, 10.0.2.2:47500 -> 10.0.2.15:6443 tcp SYN
+
+.. warning::
+
+    Make sure that none of the communications required to access the cluster or
+    for the cluster to work properly are denied. They should appear as ``action
+    allow``.
+
+
```

Comment:
We miss the actual instructions for the user on how to turn this monitor output into a rule.

A couple of suggestions, either:
* Refer the user to https://docs.cilium.io/en/v1.8/gettingstarted/policy-creation/#observe-policy-verdicts
* Refactor the above doc into a generic enough piece that it could be reused for the host endpoint.

Reply:
I'll refer users to `Creating policies from verdicts` (btw, we're not consistent in our title capitalization across the docs).

There's one bit that's probably more common with the host firewall, without being specific to the host firewall: its policies tend to rely more on entities, mostly because there's a lot more sources and destinations to allow. Maybe it would make sense to extend `Creating policies from verdicts` with one example of an entity-based rule?

Reply:
I guess I'd prefer to put such logic here to keep the creating policies GSG as simple as possible, but if there's a minimal recommendation to make either in that creating policies GSG or in this doc here then sure. It probably makes sense to at least mention that entities policies can be useful for enforcement(+link). This isn't a panacea, we won't cover all possible options, ultimately we still leave it up to the user to understand the policy model / what's available and craft their policies using the constructs we have available.